src/worker/exporters/nvidia_exporter.py

```diff
@@ -276,15 +276,40 @@ def Loop(self):
             pass
 
 
+def get_custom_config():
+    try:
+        with open('/tmp/moneo-worker/moneo_config.json') as f:
+            mon_config = json.load(f)
+
+        sample_per_min = int(mon_config['exporter_conf']['gpu_sample_interval'])
```

Comment:
jq is used earlier to get JSON values, and this notation here. Should we be consistent in what we use?

Reply:
Good observation. The reasoning is because the jq method is used in a bash script. This is in python, so I used a more native way to transfer contents directly into a dictionary vs making a shell call (for jq) from python and parsing the contents then putting them in a dictionary manually.